### PR TITLE
Update autos-and-locals-windows.md

### PR DESCRIPTION
**Pull request by HeMinzhang**

### CODE REVIEW

**Comment by HeMinzhang** [comment]
I don't know why you would accept a pull with a document exception.

File: docs/debugger/autos-and-locals-windows.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```
Comment (HeMinzhang) [inline docs/debugger/autos-and-locals-windows.md]: a pull with a document exception.

**Comment by HeMinzhang** [comment]
qq 20190116111803

**Comment by LisandroSu** [comment]
Hello, @HeMinzhang 

Thank for your contribution. We are currently reviewing it. We will communicate the decision taken  accordingly. 
 
Thanks for your time and support on this.

Kind regards,

Microsoft C&E Localization Team

**Comment by HeMinzhang** [comment]
Sorry, my English is very poor, I don't know how to express it.

**Comment by agschi** [comment]
@HeMinzhang 

Thank you for your feedback and suggestion!

The suggestion will be merged to the production, we will re-release this articles in the next loc release round.

Microsoft C&E Localization Team

**Pull request merged**